src/form/CardListForm.tsx

```diff
@@ -0,0 +1,55 @@
+import {useContext} from "react";
+import CardContext from "../context/CardContext.tsx";
+import Display from "../components/Display.tsx";
+import DivButton from "../components/DivButton.tsx";
+
+function CardListForm() {
+    const {cards} = useContext(CardContext)
+
+    const formattedCardNumber = (first: string, second: string) => {
+        return `${first}-${second}-****-****`
+    }
+
+    return (
+        <div className="root">
+            <div className="app flex-column-center">
+                <div className="flex-center">
+                    <h2 className="page-title mb-10">보유 카드</h2>
+                </div>
+                {cards
+                    .slice() // 배열을 복사하여 원본 배열에 영향을 주지 않도록 함
+                    .sort((a, b) => b.id - a.id) // id를 내림차순으로 정렬
```

Comment:
a: [toSorted](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)와 동일합니다.
```suggestion
                {cards
                    .toSorted((a, b) => b.id - a.id) // 배열을 복사 및 id를 내림차순으로 정렬
```